### ISSUE TITLE
sparkle feed download url broken for all versions.

### ISSUE THREAD

**Issue by peetinc** [rxhanson/Rectangle#73]
from https://rectangleapp.com/downloads/updates.xml -

`<enclosure url="https://rectangleapp.com/downloads/Rectangle0.19.dmg" sparkle:version="23"
                       sparkle:shortVersionString="0.19" length="10684439" type="application/octet-stream"
                       sparkle:edSignature="MvhhFfprPjcLE8lWktOIHqOHRiJwgQKKetjOG0EPEiF36ApMxc36FND7vsT5B+5kkpHL8cHYd4vw6X6NYSK3BA=="/>`

https://rectangleapp.com/downloads/Rectangle0.19.dmg appears to be dead. As are any of they previous versions. 

Thanks for the handy app.

**Comment by rxhanson** [comment]
See https://github.com/rxhanson/Rectangle/issues/56 and https://github.com/rxhanson/Rectangle/issues/70

**Comment by peetinc** [comment]
I was very much hoping to use the feed for an AutoPkg as the Sparkle feed is usually the source of authority for releases.  Using the GitHub release feed is another option, but one that does require writing the recipe by hand instead of using Recipe Robot. (As an aside Recipe Robot creates better and safer recipes than most admins write by hand.)

Out of curiosity, why are you limiting this to the Sparkle user agent? Update tracking? Reducing the load rectangleapp.com? 

Thanks a million for the response and your work on the app.

**Comment by rxhanson** [comment]
> Using the GitHub release feed is another option, but one that does require writing the recipe by hand instead of using Recipe Robot. (As an aside Recipe Robot creates better and safer recipes than most admins write by hand.)

Looks like Recipe Robot has an option to use a GitHub url? I didn't read into it to see if that would work for this or not. 

> Out of curiosity, why are you limiting this to the Sparkle user agent? Update tracking? Reducing the load rectangleapp.com?

Reducing load on rectangleapp.com ($). It's a free app. I don't track users. I've been approved for GitHub Sponsors, so I might set that up and open up the Sparkle feed depending on how things go.

**Comment by peetinc** [comment]
> Looks like Recipe Robot has an option to use a GitHub url? I didn't read into it to see if that would work for this or not.

Yeah it works for apps without Sparkle feeds. If you feed Recipe Robot the GitHub url, it downloads and inspects the app for a Sparkle feed then tries to use it to write the recipe with the ideal that the Sparkle feed is the source of authority. 

> Reducing load on rectangleapp.com ($). It's a free app. I don't track users. I've been approved for GitHub Sponsors, so I might set that up and open up the Sparkle feed depending on how things go.

Totally understood. Is there anything policy or technically blocking you from direction Sparkle feed to the GitHub URL to save on that bandwidth? https://github.com/rxhanson/Rectangle/releases/download/v0.19/Rectangle0.19.dmg instead of https://rectangleapp.com/downloads/Rectangle0.19.dmg?

Really no need to make a change, but mostly just curious.

**Comment by rxhanson** [comment]
I looked into it a long time ago and there was a reason I didn't do it before but I don't remember it now. I'll take another look.

**Comment by rxhanson** [comment]
I got a chance to revisit this, and the reasons for the path taken were mainly that I already had an automated infrastructure in place. Additionally, I had been wanting to add in the delta updating in sparkle, and it seemed a little out of place to always upload the deltas to github (although I'm sure it would work fine putting them there). After implementing the deltas on rectangleapp.com, I've decided I'll see how that goes with bandwidth usage, and if I run into issues then I'll try switching over to pointing sparkle to github. 

Long story short, I've removed the user agent restriction on the rectangleapp.com sparkle feed.

**Comment by peetinc** [comment]
Thank You! My AutoPkg recipes work now without any added trickery.
